modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/syncml/impl/SyncmlServiceImpl.java

```diff
@@ -346,12 +344,6 @@ private boolean enrollDevice(Document request) throws WindowsDeviceEnrolmentExce
                 }
                 Device generateDevice = generateDevice(DeviceManagementConstants.MobileDeviceTypes.
                         MOBILE_DEVICE_TYPE_WINDOWS, devID, modVersion, imsi, imei, devMan, devMod, user);
```

Comment:
fixed the issue by creating bean class for windowsDevice.
https://github.com/wso2/product-mdm/pull/154